library/std/math.qs

```diff
@@ -803,10 +797,7 @@ namespace Microsoft.Quantum.Math {
         Fact(n >= 0, "The factorial is not defined for negative inputs.");
         Fact(n <= 20, "The largest factorial that can be stored as an Int is 20!. Use FactorialL or ApproximateFactorial.");
 
-        [1, 1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800,
-        39916800, 479001600, 6227020800, 87178291200, 1307674368000,
-        20922789888000, 355687428096000, 6402373705728000,
-        121645100408832000, 2432902008176640000][n]
+        [1, 1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800, 39916800, 479001600, 6227020800, 87178291200, 1307674368000, 20922789888000, 355687428096000, 6402373705728000, 121645100408832000, 2432902008176640000][n]
```

Comment:
This string is very long with plenty of opportunities to split it. Are you sure it's the best way to format it?

Reply:
The formatter as it is cannot reason about the length of lines, so there is no way for it to know that this line needs to be split or where to split it.

Reply:
What about splitting of a similar array later in this file?

Reply:
Whether or not all the elements of a sequence like this are separated by a space or by newlines is determined by whether or not there is a newline present in the whitespace between the opening delimiter and the fist item. We could change this array to be formatted vertically by adding a newline between the `[` and the first `1`.

Reply:
This does look much better formatted vertically (in my opinion). I've applied that change here.